bgpd/bgp_fsm.c

```diff
@@ -2055,3 +2256,498 @@ int bgp_event_update(struct peer *peer, int event)
 
 	return ret;
 }
+/* BGP GR Code */
+
+int bgp_gr_lookup_n_update_all_peer(struct bgp *bgp,
+			enum global_mode global_new_state,
+			enum global_mode global_old_state)
+{
+	struct peer *peer = {0};
+	struct listnode *node = {0};
+	struct listnode *nnode = {0};
+	enum peer_mode  peer_old_state = PEER_INVALID;
+
+	for (ALL_LIST_ELEMENTS(bgp->peer, node, nnode, peer)) {
+
+		if (BGP_DEBUG(graceful_restart, GRACEFUL_RESTART))
+			zlog_debug(
+			"%s BGP_GR:: Peer: (%s) :",
```

Comment:
We use `[]` somewhere to identify specific event, maybe we can use `[BGP_GR]` here instead ::?